pkg/cloudprovider/vsphereparavirtual/instances.go

```diff
@@ -31,7 +31,7 @@ import (
 	cloudprovider "k8s.io/cloud-provider"
 	"k8s.io/klog/v2"
 
-	vmopv1alpha1 "github.com/vmware-tanzu/vm-operator-api/api/v1alpha1"
+	vmopv1alpha2 "github.com/vmware-tanzu/vm-operator/api/v1alpha2"
```

Comment:
We usually name the alias `vmopv1` no matter it is v1alpha1 or v1alpha2. You don't need to refactor the alias name next time we bump the vmop version to v1alpha3 or v1

Reply:
Make sense, thanks @DanielXiao